server/base/src/main/java/org/apache/accumulo/server/ServerConstants.java

```diff
@@ -68,7 +72,7 @@
   public static final int LOGGING_TO_HDFS = 4;
   public static final BitSet CAN_UPGRADE = new BitSet();
   static {
-    for (int i : new int[] {DATA_VERSION, MOVE_TO_ROOT_TABLE, MOVE_DELETE_MARKERS, LOGGING_TO_HDFS}) {
+    for (int i : new int[] {DATA_VERSION, MOVE_TO_REPLICATION_TABLE, MOVE_TO_ROOT_TABLE, MOVE_DELETE_MARKERS, LOGGING_TO_HDFS}) {
```

Comment:
1.6 and 1.7 to 1.8 sound like they should both work to me.

My gut reaction is to not worry about 1.5, but is there any fundamental problem in supporting that (is it just a testing burden)? If we don't support a direct upgrade from 1.5, what's the fail-safe? Could users just bulk-import the old files into a new instance?